snippets/lvms-disconnected-ImageSetConfig.adoc

```diff
@@ -0,0 +1,36 @@
+:_content-type: SNIPPET
+.Example ImageSetConfiguration file for {lvms}
+[source,yaml,subs="attributes+"]
+----
+kind: ImageSetConfiguration
+apiVersion: mirror.openshift.io/v1alpha2
+archiveSize: 4 <1>
+storageConfig: <2>
+  registry:
+    imageURL: example.com/mirror/oc-mirror-metadata <3>
+    skipTLS: false
+mirror:
+  platform:
+    channels:
+    - name: stable-{product-version} <4>
+      type: ocp
+    graph: true <5>
+  operators:
+  - catalog: registry.redhat.io/redhat/redhat-operator-index:v{product-version} <6>
+    packages:
+    - name: lvms-operator <7>
+      channels:
+      - name: stable <8>
+  additionalImages:
+  - name: registry.redhat.io/ubi8/ubi:latest <9>
+  helm: {}
+----
+<1> Add `archiveSize` to set the maximum size, in GiB, of each file within the image set.
+<2> Set the back-end location to save the image set metadata to. This location can be a registry or local directory. It is required to specify `storageConfig` values, unless you are using the Technology Preview OCI feature.
+<3> Set the registry URL for the storage backend.
+<4> Set the channel to retrieve the {product-title} images from.
+<5> Add `graph: true` to generate the OpenShift Update Service (OSUS) graph image to allow for an improved cluster update experience when using the web console. For more information, see _About the OpenShift Update Service_.
+<6> Set the Operator catalog to retrieve the {product-title} images from.
+<7> Specify only certain Operator packages to include in the image set. Remove this field to retrieve all packages in the catalog.
+<8> Specify only certain channels of the Operator packages to include in the image set. You must always include the default channel for the Operator package even if you do not use the bundles in that channel. You can find the default channel by running the following command: `oc mirror list operators --catalog=<catalog_name> --package=<package_name>`.
```

Comment:
Suggest putting the kinda-long command on its own in a block:

```
... You can find the default channel by running the following command: 
+
[source,terminal]
----
$ oc mirror list operators --catalog=<catalog_name> --package=<package_name>
----
```

Reply:
It messes up the callout numbering and I'm not sure if there's a solution to resolve this as you'd suggested so I didn't implement this one. Everything else is implemented, please merge.

Reply:
Oh yeah, I forgot about how it doesn't work when the block is part of a procedure/step. :( E.g., the numbering is fine if this were just a code block example on its own, not in a step:

<img width="807" alt="image" src="https://user-images.githubusercontent.com/3442316/234363051-19037c41-70c1-4d27-a3aa-cc5235b08ae7.png">

Well, thanks for trying! 😅